scripts/tls-files/client.pem

```diff
@@ -0,0 +1,47 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Should we really have sample certs/keys in scripts/tls? Not sure why we wouldn't be generating these for tests.

Reply:
I think we could. The only trouble I have at the moment is that the tool _can only_ be used as an executable. Unless we make it an independent module, we can't define a `library` for it in the cardano-tools `.cabal` file.